workflow/scripts/read_filter.py

```diff
@@ -4,18 +4,27 @@
 import sys
 from Bio import SeqIO
 import gzip
+import shutil
 
 # start logging
 sys.stderr = sys.stdout = open(snakemake.log[0], "wt")
 
+
+def is_zipped(fname):
+    return fname.endswith(".gz")
+
```

Comment:
We also need a sanity check, raise a WorkflowError, when

- `snakemake.config["min_length"]` not convertible into an integer.